diff --git a/segments/weather.sh b/segments/weather.sh
index c4031c9..ce0f9e4 100755
--- a/segments/weather.sh
+++ b/segments/weather.sh
@@ -2,7 +2,7 @@
 # Prints the current weather in Celsius, Fahrenheits or lord Kelvins. The forecast is cached and updated with a period of $update_period.
 
 # You location. Find a string that works for you by Googling on "weather in <location-string>"
-location="Lund, Sweden"
+location="Tokyo, Japan"
 
 # Can be any of {c,f,k}.
 unit="c"
